spec/models/your_details_form_spec.rb

```diff
@@ -1,7 +1,33 @@
 require 'spec_helper'
 
 describe Wpcc::YourDetailsForm, type: :model do
-  subject { described_class.new(age: 32, gender: 'female', salary: 35_000) }
+  let(:salary)                  { 35_000 }
+  let(:salary_frequency)        { 'year' }
+  let(:contribution_preference) { 'full' }
+
+  describe '.load_config' do
+    context 'file exists' do
```

Comment:
the implication with this context is the opposing context e.g. file does not exist. We need to handle that in some way. Potentially, a different user story, but we need to ensure it does not get lost.